src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,79 @@
 package core.basesyntax;
 
+import java.io.*;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final String COMMA_SEPARATOR = ",";
+    private int result = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFromFile(fromFileName);
+        writeToFile(toFileName, writeToFile());
+    }
+
+    private static String readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        StringBuilder builder = new StringBuilder();
+
+        try {
+        BufferedReader reader = new BufferedReader(new FileReader(file));
+        String value = null;
+
+            value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append("|");
+                value = reader.readLine();
+            }
+
+        }catch (IOException e) {
+            throw new RuntimeException("can`t read file...", e);
+        }
+            return builder.toString();
+        }
 
+    private int[] countData(int[] builder) {
+        int supply = 0;
+        int buy = 0;
+        String[] res = builder.toString().split("|");
+        for (String a: res) {
+            String[] temporary = a.split(",");
+            if (temporary[0].equals("supply")) {
+                supply += Integer.valueOf(temporary[1]);;
+            } else if (temporary[0].equals("buy")) {
+                buy += Integer.valueOf(temporary[1]);
+            }
```

Comment:
you have constants, why don't you use them?
also, make array's indexes constant as well